core/src/main/scala/org/apache/spark/scheduler/DAGScheduler.scala

```diff
@@ -1357,7 +1345,6 @@ class DAGScheduler(
       logDebug("Additional executor lost message for " + execId +
                "(epoch " + currentEpoch + ")")
     }
```

Comment:
Is it necessary to submit some newly-waiting stages here (e.g., if shuffle output was lost for a map stage, so now that map stage needs to be re-run)?